lstmcpipe/onsite_mc_merge_and_copy_dl1.py

```diff
@@ -178,9 +178,14 @@ def main(input_dir, flag_full_workflow=False, particle2jobs_dict={}, particle=No
             # 3.1 sbatch the jobs (or send them interactively depending) if the script is(not) run as part of the
             # whole workflow
             # filelist = [os.path.join(tdir, f) for f in os.listdir(tdir)]
-
-            cmd = f"lstchain_merge_hdf5_files -d {tdir} -o {output_filename} --no-image {flag_no_image} " \
-                  f"--smart {flag_merge}"
+            if workflow_kind == 'lstchain' or workflow_kind == 'hiperta':
+                cmd = f"lstchain_merge_hdf5_files -d {tdir} -o {output_filename} --no-image {flag_no_image} " \
+                f"--smart {flag_merge}"
+            else:
+                if flag_no_image:
+                    cmd = f'ctapipe-merge --input-dir {tdir} --output {output_filename} --skip-images {flag_no_image} --skip-simu-images {flag_no_image}'
+                else:
+                    cmd = f'ctapipe-merge --input-dir {tdir} --output {output_filename}'
```

Comment:
Unfortunately not.
`skip-images` is a flag in the tool, it doesnt take an argument. Alternatively you could write `--MergeTool.skip_images=True/False`

Reply:
Ah right !
Then as you prefer for the implementation :)

Reply:
It might not be super well-known that you can pass arguments to the underlying tools directly and I dislike mixing both ways (not using `--MergeTool.input_dir` as well), but it really doesn't matter here. Whatever you think is the most readable

Reply:
Oh, but then adding the bool variable is wrong

Reply:
Please use lists of options for `subprocess`, that will ensure correct quoting. It also makes it easy to add flags conditionally:

```
cmd = ["ctapipe-merge", f"--input-dir={tdir}", ...]
if flag_no_image:
    cmd.append("--skip-images")
suprocess.run(cmd)
```

Reply:
Actually `subprocess` is not even used here, its `os.system`, which does not take a list (?)
I suppose switching to subprocess is preferable, but that would affect all the other scripts as well

Reply:
os.system should not be used, see the comment in the docs here: https://docs.python.org/3/library/os.html#os.system

> The subprocess module provides more powerful facilities for spawning new processes and retrieving their results; using that module is preferable to using this function. See the Replacing Older Functions with the subprocess Module section in the subprocess documentation for some helpful recipes.

But replacing it here, does  not mean it needs to be replaced everywhere at once.

Reply:
I would also suggest changing this in another PR...

Reply:
And leaving the call of the ctapipe tool as it is for the moment.